integration_tests/commands/resp/getwatch_test.go

```diff
@@ -101,10 +105,13 @@ func TestGETWATCHWithSDK(t *testing.T) {
 		watch := subscriber.client.WatchConn(context.Background())
 		subscribers[i].watch = watch
 		assert.True(t, watch != nil)
-		firstMsg, err := watch.Watch(context.Background(), "GET", getWatchKey)
+
+		uuid := uuid.New().String()
+		firstMsg, err := watch.Watch(context.Background(), "GET", getWatchKey, uuid)
 		assert.Nil(t, err)
-		assert.Equal(t, firstMsg.Command, "GET")
+		assert.Equal(t, uuid, firstMsg.Command)
```

Comment:
Since uuid is generated by SDK, we don't need to verify its value - we can just verify that it's not empty.